src/azure-cli/azure/cli/command_modules/rdbms/_transformers.py

```diff
@@ -142,3 +142,17 @@ def table_transform_output_parameters(result):
         table_result.append(new_entry)
 
     return table_result
+
+
+def transform_backup(result):
+    if result.type == 'Microsoft.DBforMySQL/flexibleServers/backups' and len(result.name) > 62:
+        result.name = result.name[62:]
```

Comment:
Is it better to ask server to return the exact name instead of trim from client side?

Reply:
I'm replicating what portal does:

> There will be need to extract the customer provided backup name from the long-generated backup name stored in the backend and display the same.

Reply:
If all the clients have to extract, then it's necessary for server side to do the extraction and save all clients including CLI, Powershell, Portal...